proc-macros/src/rpc_macro.rs

```diff
@@ -43,12 +46,15 @@ pub struct RpcMethod {
 }
 
 impl RpcMethod {
-	pub fn from_item(mut method: syn::TraitItemMethod) -> Result<Self, syn::Error> {
-		let attributes = attributes::Method::from_attributes(&method.attrs).respan(&method.attrs.first())?;
+	pub fn from_item(attr: Attribute, mut method: syn::TraitItemMethod) -> syn::Result<Self> {
+		let [aliases, name] = AttributeMeta::parse(attr)?.retain(["aliases", "name"])?;
+
+		let aliases = parse_aliases(aliases)?;
+		let name = name?.string()?;
```

Comment:
I think I'm finally happy with the proc-macro-inside-proc-macro-less `bae` replacement here.